src/plugins/intel_cpu/src/transformations/tpp/x64/pass/fuse_tpp_to_equations.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/itt.hpp"
+#include "transformations/tpp/x64/op/eltwise.hpp"
+#include "fuse_tpp_to_equations.hpp"
+#include "snippets/utils/utils.hpp"
+#include "transformations/tpp/x64/op/equation.hpp"
+#include "snippets/lowered/port_descriptor.hpp"
+#include "snippets/pass/hash.hpp"
```

Comment:
`"snippets/pass/hash.hpp"` is extra and can be removed

```suggestion
#include "fuse_tpp_to_equations.hpp"

#include "transformations/tpp/x64/op/eltwise.hpp"
#include "transformations/tpp/x64/op/equation.hpp"
#include "snippets/lowered/port_descriptor.hpp"
#include "snippets/utils/utils.hpp"
#include "snippets/itt.hpp"
```

Reply:
Right, removed

Reply:
Discussed offline: let's specify preferred order of includes in cpp files and share it with the team